### DISCUSSION TITLE
Trying to understand why in Langfuse Trace under Stream Completion I get this output:

### DISCUSSION THREAD

**Discussion by howtodoml** [langfuse/langfuse#2401] [Support]
Trying to understand why in Langfuse Trace under Stream Completion I get this output:
not serializable object of type: LangfuseResponseGeneratorSync

Can anybody please explain?

_Originally posted by @howtodoml in https://github.com/langfuse/langfuse/discussions/2348_
This happens with langfuse python package version upgrade to 2.35.2
Setting it back to 2.29.2 resolves this issue
This is the code snippet that generates this:
@observe(as_type="generation")
def get_completion_stream(messages,session_id,**kwargs):
    langfuse_context.update_current_observation(name="Stream Completion",session_id=session_id)
    return openai_client.chat.completions.create(
        name="stream-completion",
        model=AZURE_OPENAI_MODEL,
        messages = messages,
        temperature=float(AZURE_OPENAI_TEMPERATURE),
        max_tokens=int(AZURE_OPENAI_MAX_TOKENS),
        top_p=float(AZURE_OPENAI_TOP_P),
        stop=AZURE_OPENAI_STOP_SEQUENCE.split("|") if AZURE_OPENAI_STOP_SEQUENCE else None,
        stream=SHOULD_STREAM,
        seed=int(AZURE_OPENAI_SEED)
    )
    
    
![image](https://github.com/langfuse/langfuse/assets/103862974/54dd542b-7c8a-4975-86d6-e6a0e29007ac)

**Comment by marcklingen** [comment]
Langfuse automatically logs outputs of functions and logs this error if the return value is not serializable. You can override the output via update_current_observation or return a value that is serializable

**Reply by howtodoml** [reply]
As mentioned above in the issue 
This happens with langfuse python package version upgrade to 2.35.2
Setting it back to 2.29.2 resolves this issue.
So to me it is a change in behaviour with version 2.35.2.  Please advise.
